cloud/filestore/libs/vfs_fuse/vhost/fuse_virtio.c

```diff
@@ -470,6 +470,8 @@ void virtio_session_exit(struct fuse_session* se)
         sched_yield();
     }
 
+    se->exited = 1;
```

Comment:
before this change, we've set the se->exited after the thread exit
now when setting it from Session.Exit() ->fuse_session_exit() we setting it when the thread is still potentially running.
Let's double check that there is no logic relying on this invariant (setting se->exited after all loop threads completed)